src/scss/utils/theme-styles.scss

```diff
@@ -23,8 +23,8 @@ $default-theme-colors: (
   "purple": #6239C8,
   "blue": #1F6BFE,
   "dark-blue": #123581,
-  "default-accent": $primary,
-  "default-background": $dark,
+  "default-accent": $theme-bg-lighter,
+  "default-background": $theme-bg,
```

Comment:
Just interested in whether there was a specific reason for switching these variables out?